app/views/layouts/application.html.erb

```diff
@@ -28,11 +28,14 @@
 <body>
 <%= render 'shared/skip_links' %>
 <%= render 'shared/header' %>
-<%= render 'shared/alerts' %>
 <%= yield :breadcrumbs %>
 
 <main role="main" id="main">
   <div class='l-constrained'>
+    <div class="l-flashes">
```

Comment:
We alluded to this in the main PR message above (although granted it's not clear). Basically we're having to wrap the flashes and notifications in a div just now for alignment with the logo and the breadcrumbs, which are getting their styles from frontend. This div is a temporary splint until we deal with the root cause. Ideally we don't want to make that whole fix in this PR, as it is already pretty big.

It's a bit complicated to explain what all we think needs to be done in a comment, but perhaps we can demo this to you at the frontend meeting today?